sorc/global_cycle.fd/cycle.f90

```diff
@@ -7,40 +7,33 @@
 !!  The surface update component runs with threads.  The NSST
 !!  update component in not threaded.
 !!
-!!  The program can be run in the following ways:
-!!  1) Update the surface fields only.  NSST fields are not
-!!     processed.  Invoke this option by setting namelist
-!!     variable DONST=.false.  Output files only contain
-!!     surface fields.
+!!  There are three main options (which can be called in combination): 
+!!  1) Update the surface fields with sfccylce (do_sfccycle = .true.) 
+!!  2) Update the land states with increments read in from file (do_lndinc = .true.) 
+!!  3) Update the NSST field, several options: 
 !!
-!!  2) Update the surface fields and NSST TREF field using
```

Comment:
Can you use "2." instead of "2)"? And the same for the other numbers. Then doxygen will understand you mean a numbered list.

Reply:
Sure. I'll update that now.

Reply:
Done.

Reply:
Ed, can you please mark this change as done? I have a message saying merging is blocked until this change is made.